### PR TITLE
TM-4134, add timestamp to created and modified by agenda item

### PR DESCRIPTION
**Pull request by M00staff**
Ended up going with this
image

Dual Merge: 
- [API PR](https://github.com/MetaPhase-Consulting/State-TalentMAP-API/pull/771)
- [Mock PR](https://github.com/MetaPhase-Consulting/mock-fsbid/pull/353)

### CODE REVIEW

File: src/sass/_panelMeetingAgendas.scss [hunk 1]
```diff
@@ -335,4 +335,7 @@
 .ai-updater-creator {
   min-width: 300px;
   font-size: 16px;
+  .date {
+    font-size: 1.2rem;
```
Comment (SidelineCory24) [inline src/sass/_panelMeetingAgendas.scss:339]: lets choose px instead here

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 2]
```diff
@@ -39,7 +39,9 @@ const AgendaItemRow = props => {
 
   const agendaStatus = get(agenda, 'status_short') || 'None Listed';
   const updaterMiddleInitial = get(agenda, 'updaters.middle_name', '')?.slice(0, 1) || 'NMN';
+  const updaterDate = (get(agenda, 'creator_date', null));
   const creatorMiddleInitial = get(agenda, 'creators.middle_name', '')?.slice(0, 1) || 'NMN';
+  const creatorDate = (get(agenda, 'modifier_date', null));
```
Comment (SidelineCory24) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:44]: is there a reason for the outer parentheses?

Comment (M00staff) [reply src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:44]: yes - there was... the reason was typo! 😅

**Comment by SidelineCory24** [comment]
@elizabeth-jimenez @scott062 will we want a clearer fallback for the date if it does not appear? Normally have context and show none listed, `ie: TED: None listed`, but here we won't show the date at all.

image

suggesstion
image

**Comment by elizabeth-jimenez** [comment]
> @elizabeth-jimenez @scott062 will we want a clearer fallback for the date if it does not appear? Normally have context and show none listed, `ie: TED: None listed`, but here we won't show the date at all.
> 
> image
> 
> suggesstion (open to a new fallback text) image

Good point! Yea, we should probably go with 'None' since, I know we are short on space or maybe --/--/-- ?!

**Comment by M00staff** [comment]
i went with the dashes - it'll take up the same space, I think it looks better this way if one date is there and the other isn't
image

**Comment by scott062** [comment]
The line feels funky with the different font size - two suggestions:
A. Move date to second line (either indented slightly or all the way to the right)
B. Make some logical spacing between name and date (same line) so the fields are more aligned (Aligned Left - Name, Aligned Right - Date). This might be harder to do b/c of name length variation. 
-
Or any other ideas?

**Comment by M00staff** [comment]
> The line feels funky with the different font size - two suggestions:
> 
> A. Move date to second line (either indented slightly or all the way to the right)
> 
> B. Make some logical spacing between name and date (same line) so the fields are more aligned (Aligned Left - Name, Aligned Right - Date). This might be harder to do b/c of name length variation.
> 
> Or any other ideas? Maybe i'm just hung up on this and its not as big of a deal

it's definitely not worth getting hung up on - but i agree that it doesn't look great.  (hopefully soon we can get UX input on all this from Tim)

ill do one more pass over and see if i can get it looking decent

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 3]
```diff
@@ -38,7 +38,9 @@ const AgendaItemRow = props => {
   const cdo = get(clientData, 'cdo.name') || 'None Listed';
 
   const agendaStatus = get(agenda, 'status_short') || 'None Listed';
+  const updateDate = get(agenda, 'modifier_date', null);
   const updaterMiddleInitial = get(agenda, 'updaters.middle_name', '')?.slice(0, 1) || 'NMN';
+  const createDate = get(agenda, 'creator_date', null);
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:43]: you can just default to ur fallback in here `const updateDate = get(agenda, 'modifier_date') ? {formatDate(get(agenda, 'modifier_date'), 'MM/DD/YY')}` : ' --/--/--';` and then just show updateDate in the span

Comment (elizabeth-jimenez) [reply src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:43]: same for created

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx [hunk 4]
```diff
@@ -121,8 +123,18 @@ const AgendaItemRow = props => {
               }
             </div>
             <div className="ai-updater-creator">
-              <div className="label">Created By: <span>{get(agenda, 'creators.last_name' || '')}, {get(agenda, 'creators.first_name' || '')} {creatorMiddleInitial}</span></div>
-              <div className="label">Modified By: <span>{get(agenda, 'updaters.last_name' || '')}, {get(agenda, 'updaters.first_name' || '')} {updaterMiddleInitial}</span></div>
+              <div className="label">
+                <span>
+                  Created: {get(agenda, 'creators.last_name' || '')}, {get(agenda, 'creators.first_name' || '')} {creatorMiddleInitial}
```
Comment (elizabeth-jimenez) [inline src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:128]: the or should go outside of the get 😁

Comment (elizabeth-jimenez) [reply src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx:128]: it was already like that, but not sure why 🤔

**Comment by M00staff** [comment]
image

File: src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx (hunk 3 above)
Comment (M00staff) [reply src/Components/Agenda/AgendaItemRow/AgendaItemRow.jsx]: VERY slick! 😎

**Review by elizabeth-jimenez — APPROVED** [review]

**Review by SidelineCory24 — APPROVED** [review]
Looks good to go!🚀

**Pull request merged**